bin/update-readmes.js

```diff
@@ -17,6 +17,7 @@ const packages = [
 		'Autogenerated selectors': 'src/selectors.js',
 	} ],
 	'data',
+	'data-controls',
```

Comment:
@nosolosw, have you had a chance to revisit this list so we could use blacklist instead to make it easier to add new packages?

Reply:
It has grown on me that an allow list works better in this case for two reasons:

- I know we've talked about detecting the entry point from package.json. Note, however, how some packages will need more than this (example: `core-data`. This will require an allow list anyway.
- Using a deny list will do unnecessary extra work. Currently, docgen will report an error if it doesn't find a token in a README file, but this can be improved to gracefully continue. However, we'll need parsing the package entry point files anyway to know whether there is a token or not - wasting those cycles.

Adding/Removing a package is not a common operation, so I think is fine to require authors to do this explicitely if they want API doc generation.